resources/idlharness.js

```diff
@@ -3451,6 +3451,17 @@ IdlNamespace.prototype.test_self = function ()
 
 IdlNamespace.prototype.test = function ()
 {
+    // If the namespace object is not exposed, only test that. Members can't be
```

Comment:
This broke the `tests/functional/idlharness/IdlNamespace/test_operation.html` unit test, I think

Reply:
Fixed and added an additional test for the new functionality.